packages/plugins/users-permissions/admin/src/pages/Roles/ListPage/components/TableBody.js

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { IconButton, Tbody, Text, Tr, Td, Row } from '@strapi/parts';
+import { EditIcon, DeleteIcon } from '@strapi/icons';
+import { CheckPermissions } from '@strapi/helper-plugin';
+import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+
+import { getTrad } from '../../../../utils';
+import pluginId from '../../../../pluginId';
+
+const TableBody = ({
+  sortedRoles,
+  canDelete,
+  permissions,
+  setRoleToDelete,
+  setShowConfirmDelete,
+  showConfirmDelete,
```

Comment:
Can't these two one be gathered into the same prop an be called `onDeleteRole` ?

Reply:
That actually crossed my mind not sure why I didn't do it that way in the end! I'll update.